ratatoskr/app/templates/app/pages/create_timeslots.html

```diff
@@ -3,16 +3,64 @@
 {% block title %}Create Timeslots{% endblock title %}
 
 {% block body %}
+    {% if errors %}
+        <div class="alert alert-danger" role="alert">
+          {{ errors }}
+        </div>
+    {% endif %}
     <div class="d-flex justify-content-center align-items-center h-75">
-        <div class="card w-50 h-50">
+        <div class="card w-75 h-50">
             <div class="card-header text-center">
                 <h2 class="mt-2">Create Timeslots</h2>
             </div>
             <div class="card-body">
-                <form method="POST">
+                <form method="POST" v-scope="{ multipleTimeslots: true }">
                     {% csrf_token %}
-                    {{ form.as_p }}
-                    <input class="btn btn-primary" type="submit">
+                    <div class="row g-2">
+                        <div class="mb-3 col">
+                            <label for="from_date" class="form-label">Date From</label>
+                            <input type="date" class="form-control" id="from_date" name="from_date">
+                        </div>
+                        <div class="mb-3 col">
+                            <label for="to_date" class="form-label">Date To</label>
+                            <input type="date" class="form-control" id="to_date" name="to_date">
+                        </div>
+                    </div>
+                    <div class="row g-2">
+                        <div class="mb-3 col">
+                            <label for="from_time" class="form-label">Time From</label>
+                            <input type="time" class="form-control" id="from_time" name="from_time">
+                        </div>
+                        <div class="mb-3 col">
+                            <label for="to_time" class="form-label">Time To</label>
+                            <input type="time" class="form-control" id="to_time" name="to_time">
+                        </div>
+                    </div>
+                    <div class="form-check form-switch mb-3">
+                      <input class="form-check-input" type="checkbox" id="multiple_timeslots" @click="multipleTimeslots = $el.checked" name="single_timeslot" checked>
+                      <label class="form-check-label" for="multiple_timeslots" v-html="multipleTimeslots ? 'Single Timeslot' : 'Multiple Timeslots'"></label>
+                    </div>
+                    <div v-bind:style="{ display: multipleTimeslots ? 'none' : 'block' }">
```

Comment:
Can be achieved with [v-show](https://vuejs.org/api/built-in-directives.html#v-show) directive.